integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonDescribeFormattedCommand.scala

```diff
@@ -44,89 +42,121 @@ private[sql] case class CarbonDescribeFormattedCommand(
-    results ++= Seq(("Database Name", relation.carbonTable.getDatabaseName, "")
-    )
-    results ++= Seq(("Table Name", relation.carbonTable.getTableName, ""))
-    results ++= Seq(("CARBON Store Path ", carbonTable.getTablePath, ""))
 
-    val tblProps = carbonTable.getTableInfo.getFactTable.getTableProperties
+    results ++= Seq(("", "", ""), ("### PROPERTY", "VALUE", "DEFAULT_VALUE"))
+
+    // Table Basic Information
+    results ++= Seq(("", "", ""), ("## Table Basic Information", "", ""))
+    results ++= Seq(("Database Name", carbonTable.getDatabaseName, ""))
+    results ++= Seq(("Table Name", carbonTable.getTableName, ""))
+    results ++= Seq(("Table Path", carbonTable.getTablePath, ""))
 
     // Carbon table support table comment
-    val tableComment = tblProps.asScala.getOrElse(CarbonCommonConstants.TABLE_COMMENT, "")
-    results ++= Seq(("Comment", tableComment, ""))
-    results ++= Seq(("Table Block Size ", carbonTable.getBlockSizeInMB + " MB", ""))
+    results ++= Seq(("Comment",
+        tblProps.asScala.getOrElse(CarbonCommonConstants.TABLE_COMMENT, ""), ""))
+
     val dataIndexSize = CarbonUtil.calculateDataIndexSize(carbonTable, false)
+    var tableDataSizeStr = "0B"
+    var tableIndexSizeStr = "0B"
+    var lastUpdateTimeStr = "NA"
     if (!dataIndexSize.isEmpty) {
-      results ++= Seq((CarbonCommonConstants.TABLE_DATA_SIZE,
-        dataIndexSize.get(CarbonCommonConstants.CARBON_TOTAL_DATA_SIZE).toString, ""))
-      results ++= Seq((CarbonCommonConstants.TABLE_INDEX_SIZE,
-        dataIndexSize.get(CarbonCommonConstants.CARBON_TOTAL_INDEX_SIZE).toString, ""))
-      results ++= Seq((CarbonCommonConstants.LAST_UPDATE_TIME,
-        dataIndexSize.get(CarbonCommonConstants.LAST_UPDATE_TIME).toString, ""))
+      val tableDataSize =
+        dataIndexSize.get(CarbonCommonConstants.CARBON_TOTAL_DATA_SIZE).toLong
+      tableDataSizeStr = Strings.formatSize(tableDataSize)
+      val tableIndexSize =
+        dataIndexSize.get(CarbonCommonConstants.CARBON_TOTAL_INDEX_SIZE).toLong
+      tableIndexSizeStr = Strings.formatSize(tableIndexSize)
+      val lastUpdateTime = dataIndexSize.get(CarbonCommonConstants.LAST_UPDATE_TIME).toLong
+      lastUpdateTimeStr = if (lastUpdateTime > 0L) {
+        new java.sql.Timestamp(lastUpdateTime).toString()
+      } else {
+        "NA"
+      }
     }
+    results ++= Seq((CarbonCommonConstants.TABLE_DATA_SIZE, tableDataSizeStr, ""))
+    results ++= Seq((CarbonCommonConstants.TABLE_INDEX_SIZE, tableIndexSizeStr, ""))
+    results ++= Seq((CarbonCommonConstants.LAST_UPDATE_TIME, lastUpdateTimeStr, ""))
 
+    // Detailed Table Properties Information
+    results ++= Seq(("", "", ""), ("## Detailed Table Properties Information", "", ""))
+    results ++= Seq(("Table Block Size", carbonTable.getBlockSizeInMB + " MB",
+        CarbonCommonConstants.BLOCK_SIZE_DEFAULT_VAL + " MB"))
     results ++= Seq(("SORT_SCOPE", tblProps.asScala.getOrElse("sort_scope", CarbonCommonConstants
```

Comment:
For these table properties, I think it is not very good to print the default value if the user does not specify. Because the default value may change from version to version, I think it is better to write the value to table property when loading the table. Then in DESC table, we can always get the table properties from the schema file